oddt/shape.py

```diff
@@ -0,0 +1,147 @@
+import numpy as np
+import scipy.stats as stats
+
+
+def standard_usr(molecule, points=[None, None, None, None]):
+    """Computes USR shape descriptor
+
+    Parameters
+    ----------
+    molecule : oddt.toolkits.ob.Molecule
+
+    points : array
+        Coordinates
+
+    Returns
+    -------
+        shape_descriptor : numpy array
+    """
+    atoms = molecule.atom_dict['coords']
+    ctd, fct, cst, ftf = points
+
+    "ctd: the molecular centroid"
+    if not ctd: ctd = atoms.mean(0)
```

Comment:
you perform the same operation for each array, so there are unnecessary repetitions in your code. use should either do it in a loop or create a single, multidimensional array and apply np.linalg.norm to it (better solution)